src/proxy.js

```diff
@@ -102,66 +169,40 @@ module.exports = (proxyTarget, host) => {
 		}
 	});
 
+	let removeElementsHTML = [ 'meta[name="robots"][content="noindex"]' ];
+	let rewriteAttributesHTML = [ 'action', 'content', 'href', 'link', 'src', 'srcset', 'style' ];
+	let rewriteElementsHTML = [ 'loc' ];
+	let rewriteRegExpsHTML = [ 'script[type="application/ld+json"]' ];
+	let harmonMiddlewareHTML = harmon([], removeElementsHTML.map(removeElement)
+		.concat(rewriteAttributesHTML.map(rewriteAttribute))
+		.concat(rewriteElementsHTML.map(rewriteElement))
+		.concat(rewriteRegExpsHTML.map(rewriteRegexp)), false);
+
+	let rewriteAttributesXML = [ 'href' ];
+	let rewriteElementsXML = [ 'loc', 'image\\:loc' ];
+	let harmonMiddlewareXML = harmon([], rewriteElementsXML.map(rewriteElement)
+		.concat(rewriteAttributesXML.map(rewriteAttribute)), false);
+
+	let rewriteAttributesXSL = [ 'href' ];
```

Comment:
sitemap.xsl file breaks if we rewrite it's `content` attribute, thus we need to separate what to parse for what files.